docs/helpers/domData.md

```diff
@@ -0,0 +1,86 @@
+@function can-stache.helpers.domData {{domData(expression)}}
+@parent can-stache.htags 16
+
+@signature `{{domData(key)}}`
+
+Uses [can-dom-data] to associate an element with the current [can-stache.scopeAndContext context].
+
+@param {String} [key] The name of the data attribute to use for the context.
+
+@signature `{{domData(key, value)}}`
+
+Uses [can-dom-data] to associate an element with a value.
+
+@param {String} [key] The name of the data attribute to use for the value.
+@param {can-stache/expressions/key-lookup|can-stache/expressions/call} [value]
+The value to associate with the key.
+
+@body
+
+## Use
+
+`{{domData(key)}}` lets you associate the current context with the element so
+the context can later be retrieved by [can-dom-data].
+
+Let’s look at a simple example:
+
+```js
+import Component from "can-component";
+import domData from "can-dom-data";
+
+Component.extend({
+  tag: "todo-list",
+  view: `
+    <ul>
+      {{#each(todos)}}
+        <li {{domData("todo")}}>
+          {{title}}
+        </li>
+      {{/each}}
+    </ul>
+  `,
+  events: {
+    "li click": function(element) {
+      const todo = domData.get.call(element, "todo");
+      // Do something with todo
+    }
+  }
+});
+```
+
+> **Note:** this contrived example is just to demonstrate the `{{domData}}`
+> helper; if you need to create a click handler, you should write
+> `<li on:click="handler(this)">` and add a `handler()` method to your
+> view-model.
```

Comment:
Is this distracting or useful? Maybe I could include a better example that wouldn’t need this disclaimer?